BackEnd/routes.js

```diff
@@ -209,6 +211,43 @@ router.get('/getTargetLocation', (req, res) => {
 	    res.send(obj.location);
 	}
     });
+
+router.post('/createAlliance', (req, res) => {
+	var jsonRequestBody = req.body;
+
+	var allianceFields = {};
+	allianceFields.allies = jsonRequestBody.allies;
+	allianceFields.targets = jsonRequestBody.targets;
+
+	var alliance = new Alliance(allianceFields);
+	var allianceId = alliance.id;
+	var allies = JSON.parse(jsonRequestBody.allies);
+
+	alliance.save(function(err) {
+		if (err) {
+			console.log("Failed to create alliance - Error: " + err.message);
+			return;
```

Comment:
When the other post requests fail, we usually return something like
res.status(400).json({
			error: 'Must have both login code and orgName specified',
		});
Can you edit your code to have a similar error message for consistency?